andela/ah-sealteam
# Ch speed mail sending 162475157

#### What does this PR do?

This PR will speed response when sending an email. It will offload email sending from the main thread so that a response can be made instantly. 

#### Description of Task to be completed?

- Emails will be sent by another thread which will ensure quick response when sending an email. This will currently affect

1.  Sending main after signup for confirmation
2. Sending email when signing up.
Will reduce sending email time from 10 seconds to 0.5 secs.
#### How should this be manually tested?
After pulling origin/develop branch from develop checkout to `ch-speed-mail-sending-162475157`
Try signing up and check response time, then try to reset the password and check response time
#### What are the relevant pivotal tracker stories?
[#162475157](https://www.pivotaltracker.com/story/show/162475157)
